core/inline.rng

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes">
+
+    <define name="libero.inline.full">
+
+        <oneOrMore>
+            <ref name="libero.inline.full.content"/>
+        </oneOrMore>
+
+    </define>
+
+    <define name="libero.inline.full.content">
```

Comment:
we'll need a name in general for what concepts like `limited` and `full` are an expression of